### PR TITLE
Support for docker-compose

### PR DESCRIPTION
**Pull request by bashbers**
Added a docker-compose basefile with some information about docker-compose inside the readme.md

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - '2306:2306/udp'
+    volumes:
+      - './missions:/arma3/mpmissions'
+      - './configs:/arma3/configs'
+      - './mods:/arma3/mods'
+      - './servermods:/arma3/servermods'
+    environment:
+      - ARMA_CONFIG=main.cfg
+      - STEAM_USER=USERNAMENOTEMAILADDRESS
+      - STEAM_PASSWORD=PASSWORD
```
Comment (Dahlgren) [inline docker-compose.yml:20]: These two could be changed to use `.env` file values instead, that's how my local dev compose file is set up at least 😅

Comment (bashbers) [reply docker-compose.yml:20]: Ah that would be great aswell, could you post an excerpt of how that is setup?

Comment (Dahlgren) [reply docker-compose.yml:20]: https://github.com/Dahlgren/Arma3Server/commit/20f18da9fbd04a4a6b44723c152e0ccd14a6932e

**Comment by bashbers** [comment]
I've added support for ENV files together with some extra details about docker compose and its usage.

**Comment by bashbers** [comment]
@BrettMayson Could this be merged please?

**Review by BrettMayson — APPROVED** [review]

**Pull request merged**